src/Components/Moormanage/Technicians/Technicians.tsx

```diff
@@ -0,0 +1,200 @@
+import { Column } from 'primereact/column'
+import { SelectButton, SelectButtonChangeEvent } from 'primereact/selectbutton'
+import { Nullable } from 'primereact/ts-helpers'
+import { Button } from 'primereact/button'
+import { Calendar } from 'primereact/calendar'
+import { TechnicianPayload, TechnicianResponse } from '../../../Type/ApiTypes'
+import { useGetTechnicianMutation } from '../../../Services/MoorManage/MoormanageApi'
+import { BillsData } from '../../../Type/CommonType'
+
+const Technicians = () => {
+  const [date, setDate] = useState<Nullable<(Date | null)[]>>(null)
+  const options: string[] = ['Open', 'Completed']
+  const [value, setValue] = useState<string>(options[0])
+  const [dataVisible, setDataVisible] = useState(false)
+  const [technicianRecord, setTechnicianRecord] = useState()
+  const [globalFilter, setGlobalFilter] = useState<string | null>(null)
+  const [selectedDate, setSelectedDate] = useState<Date | null>(new Date())
+  const [technicianData, setTechnicianData] = useState<TechnicianPayload[]>([])
+  const [filteredTechnicianData, setFilteredTechnicianData] = useState<TechnicianPayload[]>([])
+  const [getTechnicians] = useGetTechnicianMutation()
+
+  const [workOrderData, setWorkOrderData] = useState<BillsData[]>([
+    {
+      id: 0,
+      technician: 'Suncatcher',
+      techniciansName: 'John Smith',
+      dueDate: '3-12-2024',
+    },
+  ])
+
+  const technicianHeader = (
+    <div>
+      <div className="p-input-icon-left">
+        <i className="pi pi-search" />
+        <input
+          type="search"
+          value={globalFilter as string}
+          onChange={(e) => setGlobalFilter(e.target.value)}
+          placeholder="Search By name, Id..."
+          className="border-[1px] w-[30vw] p-2 pl-10 rounded-md"
+        />
+      </div>
+    </div>
+  )
+
+  const handleDateSelect = (date: Date) => {
+    setSelectedDate(date)
+  }
+
+  const handleDateUnselect = () => {
+    setSelectedDate(null)
+  }
+
+  const workOrder = (
+    <>
+      <div className="flex gap-40 rounded-md">
+        <div>
+          <p className="font-bold"> Work Orders </p>
+        </div>
+        <div>
+          <SelectButton value={value} onChange={(e) => setValue(e.value)} options={options} />
+        </div>
+      </div>
+    </>
+  )
+
+  const Billsheader = (
+    <div className="flex flex-wrap align-items-center gap-4 ">
+      <span className="text-sm font-bold text-[black]">Work Orders</span>
+      <div className=" ">
+        <div className="">
+          <SelectButton
+            style={{ fontSize: '0.2rem', fontWeight: 'bolder', height: '2rem' }}
+            value={value}
+            onChange={(e: SelectButtonChangeEvent) => setValue(e.value)}
+            options={options}
+          />
+        </div>
+      </div>
+
+      <div className="ml-72">
+        <Button
+          onClick={function (): void {
+            throw new Error('Function not implemented.')
```

Comment:
what it this ?

Reply:
Originally, this was meant for a functionality that hasn't been implemented yet. Since it's not currently in use, I removed it from the code.